test/model/deepvar_hierarchical/test_deepvar_hierarchical.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import pytest
+
+from gluonts.evaluation import backtest_metrics, MultivariateEvaluator
+from gluonts.model.deepvar_hierarchical import DeepVARHierarchicalEstimator
+from gluonts.mx.trainer import Trainer
+from generate_hierarchical_dataset import sine7
+
+
+seq_length = 100
+prediction_length = 10
+train_datasets = sine7(
+    seq_length=seq_length, prediction_length=prediction_length
+)
+
+
+@pytest.mark.parametrize(
+    "likelihood_weight, CRPS_weight, coherent_train_samples, coherent_pred_samples, warmstart_epoch_frac",
+    [
+        # Hier-E2E
```

Comment:
Just curious: Does `hybridize` work here?

Reply:
No, it does not and it is not possible I think given the problem specific not-learned tensors like projection matrix are used in the loss computation.